tasks/update.yml

```diff
@@ -0,0 +1,91 @@
+---
+# Creates a folder /opt/mattermost-updated/mattermost with the updated version
+
+- name: Remove a previously leaked mattermost-displaced folder
+  file:
+    path: /opt/mattermost-displaced
+    state: absent
+
+- name: Remove a previously leaked mattermost-updated folder
+  file:
+    path: /opt/mattermost-updated
```

Comment:
If this folder existed, it would have been removed using the earlier deploy task, so this task is unnecessary.